docs/lambda.asciidoc

```diff
@@ -1,150 +1,86 @@
-----
+include::{apm-aws-lambda-root}/docs/lambda-selector/lambda-attributes-selector.asciidoc[]
 
 [float]
-[[lambda-initialization]]
-==== Initialization
-
-It's important that the agent is started before you require *any* other modules in your Node.js application - i.e. before `http`, etc.
-
-There are two ways to achieve this.  The first is via automatic initialization using the `NODE_OPTIONS` environment variable.  The second is via manual instrumentation.
+===== Step 2: Add the APM Layers to your Lambda function
 
-==== Automatic Initialization
+include::{apm-aws-lambda-root}/docs/lambda-selector/extension-arn-replacement.asciidoc[]
+include::./lambda/nodejs-arn-replacement.asciidoc[]
 
-In order to automatically initialize the agent, you'll need to set the following environment variable in your Lambda function's `Configuration -> Environment variables` section.
+Both the {apm-guide-ref}/aws-lambda-arch.html[APM Lambda Extension] and the Node.js APM Agent are added to your Lambda function as https://docs.aws.amazon.com/lambda/latest/dg/invocation-layers.html[AWS Lambda Layers]. Therefore, you need to add the corresponding Layer ARNs (identifiers) to your Lambda function.
 
-[source,bash]
----
-NODE_OPTIONS=-r elastic-apm-node/start.js
----
+include::{apm-aws-lambda-root}/docs/add-extension/add-extension-layer-widget.asciidoc[]
 
-With this value set the agent will initialize prior to AWS loading your Lambda handler. This will allow the agent to automatically instrument your modules including the lambda handler itself.
+[float]
+===== Step 3: Configure APM on AWS Lambda
 
-==== Manual Initialization
+The APM Lambda Extension and the APM Node.js agent are configured through environment variables on the AWS Lambda function.
 
-If you can't use the `NODE_OPTIONS` variable, you'll need to manually initialize the agent and manually instrument your Lambda handler. Here's a simple lambda example with the Elastic APM agent installed:
+For the minimal configuration, you will need the _APM Server URL_ to set the destination for APM data and an _{apm-guide-ref}/secret-token.html[APM Secret Token]_.
+If you prefer to use an {apm-guide-ref}/api-key.html[APM API key] instead of the APM secret token, use the `ELASTIC_APM_API_KEY` environment variable instead of `ELASTIC_APM_SECRET_TOKEN` in the following configuration.
 
-[source,js]
-----
-// Add this to the VERY top of your lambda handler module
-const apm = require('elastic-apm-node').start({})
+include::./lambda/configure-lambda-widget.asciidoc[]
 
-exports.handler = apm.lambda(function handler (event, context, callback) {
-  callback(null, `Hello, ${event.name}!`)
-})
-----
+You can optionally <<configuration, fine-tune the Node.js agent>> or the {apm-guide-ref}/aws-lambda-config-options.html[configuration of the APM Lambda Extension].
 
-The agent will now monitor the performance of your lambda function.
+That's it; after following the steps above, you're ready to go!
```

Comment:
```suggestion
That's it! After following the steps above you're ready to go!
```

Reply:
Done in https://github.com/elastic/apm-agent-nodejs/pull/2630/commits/94da3631aaf5d09a68b93024b24c29c9f2c72f04  though I claim cut 'n paste from equiv Java and Python PRs. :)